### PR TITLE
User can set negative progress #50

### PR DESCRIPTION
**Pull request by petrenkonikita112263**
I added properties for input field. Which prevent typing negative number by arrow key, input form, manual typing and pasting.

Before
firefox_fzltswUBlF

After
firefox_SFpueuZhIe

### CODE REVIEW

File: application-task-ui/src/main/resources/TaskManager/TaskManagerClass.xml [hunk 1]
```diff
@@ -309,11 +320,12 @@
         &lt;/div&gt;
       &lt;/div&gt;
   #else
-    &lt;input class="form-control" id="TaskManager.TaskManagerClass_0_progress" name="TaskManager.TaskManagerClass_0_progress" type="number" value="${value}" /&gt;
+    &lt;input class="form-control" id="TaskManager.TaskManagerClass_0_progress" name="TaskManager.TaskManagerClass_0_progress" type="number" min="0" oninput="validity.valid||(value='');" value="${value}" /&gt;
```
Comment (trrenty) [inline application-task-ui/src/main/resources/TaskManager/TaskManagerClass.xml:323]: We could also set the upper bound to be 100. 
```suggestion
    &lt;input class="form-control" id="TaskManager.TaskManagerClass_0_progress" name="TaskManager.TaskManagerClass_0_progress" type="number" min="0" max="100" oninput="validity.valid||(value='');" value="${value}" /&gt;
```

File: application-task-ui/src/main/resources/TaskManager/TaskManagerTemplate.xml [hunk 2]
```diff
@@ -314,11 +325,12 @@
         &lt;/div&gt;
       &lt;/div&gt;
   #else
-    &lt;input class="form-control" id="TaskManager.TaskManagerClass_0_progress" name="TaskManager.TaskManagerClass_0_progress" type="number" value="${value}" /&gt;
+    &lt;input class="form-control" id="TaskManager.TaskManagerClass_0_progress" name="TaskManager.TaskManagerClass_0_progress" type="number" min="0" oninput="validity.valid||(value='');" value="${value}" /&gt;
```
Comment (trrenty) [inline application-task-ui/src/main/resources/TaskManager/TaskManagerTemplate.xml:328]: same as above

**Review by trrenty — CHANGES_REQUESTED** [review]

File: application-task-ui/src/main/resources/TaskManager/TaskManagerClass.xml (hunk 1 above)
Comment (petrenkonikita112263) [reply application-task-ui/src/main/resources/TaskManager/TaskManagerClass.xml:323]: okay, then I'll close my question (issue) #65

**Comment by petrenkonikita112263** [comment]
I have edited the title of issue, don't know is there a need to change title of PR

**Pull request merged**